README.md

```diff
@@ -22,6 +30,17 @@ To create these collections:
 npm run db:setup
 ```
 
+### CRUD Operations
+
+This server currently supports the following CRUD operations:
+
+- Read by ID to endpoint: `4_0_1/<resourceType>/<resourceId>`
+  _More functionality coming soon!_
+
+### Search
+
+The Measure Repository Server supports `Library` and `Measure` resource search by the parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#shareable-measure-repository).
```

Comment:
I could be very wrong by this but do we want to refer to this as the Measure Repository Service Server or just Measure Repository Service? Or is Measure Repository Server also correct? (sounds right, just wondering)

Reply:
Good catch! This was accidental. I think Measure Repository Service Server is probably what we want, although it is a mouthful :/